File: [CCS LABS] Windows Forms Web Image Crawler and Downloader Sample/[C#]-[CCS LABS] Windows Forms Web Image Crawler and Downloader Sample/C#/WebImageCrawler/frmMain.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.IO;
using System.Linq;
using System.Net;
using System.Text;
using System.Windows.Forms;
using System.Threading;

using DevRain.Data.Extracting;
using System.Runtime.InteropServices;



namespace WebImageCrawler
{
    public partial class Wic : Form
    {

        public Wic()
        {
            InitializeComponent();
        }

        // Checks to see if it can start.
        private void BtnGoClick(object sender, EventArgs e)
        {

            if (!string.IsNullOrEmpty(tbURL.Text))
            {
                // Change to Desktop View
                DesktopView();
            }
            else
            {
                MessageBox.Show("You need to add a valid starting URL");
            }


        }

        // Re-design the 
        private void DesktopView()
        {
            this.Hide();
            FrmDesktopView fdtv = new FrmDesktopView(tbURL.Text);
            fdtv.Show();


        }






    }
}

File: [CCS LABS] Windows Forms Web Image Crawler and Downloader Sample/[VB]-[CCS LABS] Windows Forms Web Image Crawler and Downloader Sample/VB/WebImageCrawler.ConvertedToVBNet/frmMain.vb
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.IO
Imports System.Linq
Imports System.Net
Imports System.Text
Imports System.Windows.Forms
Imports System.Threading

Imports DevRain.Data.Extracting
Imports System.Runtime.InteropServices



Public Partial Class Wic
	Inherits Form

	Public Sub New()
		InitializeComponent()
	End Sub

	' Checks to see if it can start.
	Private Sub BtnGoClick(sender As Object, e As EventArgs)

		If Not String.IsNullOrEmpty(tbURL.Text) Then
			' Change to Desktop View
			DesktopView()
		Else
			MessageBox.Show("You need to add a valid starting URL")
		End If


	End Sub

	' Re-design the 
	Private Sub DesktopView()
		Me.Hide()
		Dim fdtv As New FrmDesktopView(tbURL.Text)
		fdtv.Show()


	End Sub






End Class
